core/modules/node/node.pages.inc

```diff
@@ -883,17 +907,36 @@ function node_revision_overview($node) {
   if ((user_access('delete revisions') || user_access('administer nodes')) && node_access('delete', $node)) {
     $delete_permission = TRUE;
   }
+  $edit_permission = FALSE;
+  if ((user_access('update revisions') || user_access('administer nodes')) && node_access('update', $node)) {
+    $edit_permission = TRUE;
+  }
   foreach ($revisions as $revision) {
     $row = array();
-    if ($revision->current_vid > 0) {
-      $row[] = array('data' => t('!date by !username', array('!date' => l(format_date($revision->timestamp, 'short'), "node/$node->nid"), '!username' => theme('username', array('account' => $revision))))
-                               . (($revision->log != '') ? '<p class="revision-log">' . filter_xss($revision->log) . '</p>' : ''),
-                     'class' => array('revision-current'));
-      $row[] = array('data' => backdrop_placeholder(t('current revision')), 'class' => array('revision-current'));
+    $is_active_revision = ($revision->vid == $node->active_revision);
```

Comment:
Shouldn't we be able to call `$node->isActiveRevision()`? It also isn't clear to me where `$node->active_revision` is set.  That property is not documented anywhere that I can find.

Reply:
True we should use `isActiveRevision()` here. 
See `addField()` in the query builder in entity controller.